### PR TITLE
Tidy up the careers page

### PR DESCRIPTION
**Pull request by mruwnik**
* change url from `howcanihelp` to `how-can-i-help`
* add help urls to `routesMapper` to make it easier to change them
* components for common html stuff
* separate routes for the different pages, as this is the Remix Way©™®

### CODE REVIEW

File: app/components/DropDown/index.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ export default function DropDown({title, content}: DropDownProps) {
           className={`${isOpen ? 'rotate-90' : ''} drop-down-chevron`}
         />
       </button>
-      <div className={`drop-down-content ${isOpen ? 'open' : ''}`}>{content}</div>
+      <div className={`drop-down-content ${isOpen ? 'open' : ''}`}>{children}</div>
```
Comment (mruwnik) [inline app/components/DropDown/index.tsx:23]: this lets dropdowns be created as:
```
<Dropdown title="blablabla">
   <p>bla bla bla</p>
   <p>bla bla bla</p>
</Dropdown>
```

File: app/components/HelpGrid/index.tsx [hunk 2]
```diff
@@ -9,46 +10,46 @@ const helpItems = [
       'For both technical and non-technical roles in AI alignment, governance, and field-building',
     impact: 'Highest direct impact',
     icon: Briefcase,
-    action: '/howcanihelp/career',
+    action: helpUrl('career'),
```
Comment (mruwnik) [inline app/components/HelpGrid/index.tsx:13]: this makes it a lot easier to change urls, or at least know that they are correct(ish)

File: app/components/HowCanIHelp/Base.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <Page>
+      <div className="page-body">
+        <div className="padding-bottom-16">
+          <Link to={helpUrl()}>How can I help with AI Safety?</Link>
+        </div>
+
+        <div>
+          <h1 className="teal-500 padding-bottom-40">{title}</h1>
+
+          <HelpMethods current={subpage} />
```
Comment (mruwnik) [inline app/components/HowCanIHelp/Base.tsx:17]: this is the links to other pages at the bottom of each help page

File: app/components/HowCanIHelp/HelpItem.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import {ReactNode} from 'react'
+import LinkCard, {Link} from '~/components/LinkCard'
+
+type HelpItemProps = {
+  title?: ReactNode
+  children: ReactNode
+  links?: Link[]
+}
+const HelpItem = ({title, children, links}: HelpItemProps) => {
```
Comment (mruwnik) [inline app/components/HowCanIHelp/HelpItem.tsx:9]: this name is bad. It's basically each one of these thingies:
image

File: app/components/HowCanIHelp/Base.tsx (hunk 3 above)
Comment (zarSou9) [reply app/components/HowCanIHelp/Base.tsx:17]: Yes, much better

File: app/components/HowCanIHelp/HelpItem.tsx (hunk 4 above)
Comment (zarSou9) [reply app/components/HowCanIHelp/HelpItem.tsx:9]: Nice

**Review by zarSou9 — APPROVED** [review]
Yes, this makes much more sense. I will have to update my pr on the styling, but yeah, better if this gets merged first.

**Comment by zarSou9** [comment]
> * separate routes for the different pages, as this is the Remix Way©™®
haha

**Pull request merged**